tests/test_dockercomposeexecutor.py

```diff
@@ -18,9 +20,24 @@ def test_execute():
         ]
 
 
+def test_execute_docker_compose_v2():
```

Comment:
Instead of convoluting, the test suite this way, I'd expect the docker-compose command to be a link to the executable docker compose cli plugin, and the run the whole test suite twice with each docker-compose version. This is actually so in my Arch box
```
$ file /usr/bin/docker-compose
/usr/bin/docker-compose: symbolic link to /usr/lib/docker/cli-plugins/docker-compose
```

In the CI job, if the docker-compose binary release is simply downloaded into the /usr/bin folder instead, all tests should simply pass out-of-the-box. AFAIU this preparation will usually be distro packagers responsibility but the docker team is committed to keep it working, and in our CI job it is a burden for us because no setup-docker-compose github action can setup any compose-v2 version yet (or I haven't found it).

Then, I'd vouch to run a parallel test job that installs the docker-compose-v1 extra requirement instead so we have some guarantees that pytest-docker runs seamlessly in either compose version, until compose-v1 is deprecated in a far(?) future.

Reply:
Maybe this action https://github.com/ndeloof/install-compose-action will work? I'd like to give it a try, I'll reach you back with my findings.

Reply:
Albeit it is pretty young (just as compose v2, to be fair) and its documentation would truly help some updates... it looks rather fine!
https://github.com/n1ngu/pytest-docker/runs/5168879309
Looking into its source code, the action does pretty much what you'd expect: download the docker-compose release and place it adequately.

Use with
```
      - name: Install Docker Compose v2
        uses: ndeloof/install-compose-action@v0.0.1
        with:
          version: latest
          legacy: true
```